slack_sdk/web/internal_utils.py

```diff
@@ -231,22 +231,29 @@ def _to_0_or_1_if_bool(v: Any) -> Union[Any, str]:
 
 
 def _warn_if_text_is_missing(endpoint: str, kwargs: Dict[str, Any]) -> None:
+    missing = "text"
```

Comment:
This is just a kind of preference but having this variable at the top level of this method may be a bit safer for avoiding unassigned variable error (particularly when we add more if/elif statements in the future).

Reply:
I appreciate you taking the time to explain your thinking/reasoning here. Good things to think about (and try to remember) in the future.